### PR TITLE
nuzhy/live market subtask fixes

### PR DESCRIPTION
**Pull request by Nuzhy-Deriv**
Fixes
- Font color of texts in closed cards
- Font size of 'See all forex pairs'. (Will be updated with link component from quill once ready)
- Font size of disclaimer text (new design change)
- Position of disclaimer and 'See all forex pairs' link and gap between (new design change)

### CODE REVIEW

File: libs/components/src/lib/card/live-market/buy-sell.buttons.tsx [hunk 1]
```diff
@@ -11,10 +18,10 @@ export const BuySellButtons = ({
   return (
     <div className="flex flex-col gap-gap-md">
       <div className="flex flex-row gap-gap-md">
-        <Text size="sm" className="text-typography-default">
+        <Text size="sm" className={clsx(spreadClass(status))}>
```
Comment (prince-deriv) [inline libs/components/src/lib/card/live-market/buy-sell.buttons.tsx:21]: I think we can omit `clsx` here

**Review by prince-deriv — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/card/live-market/buy-sell.buttons.tsx [hunk 2]
```diff
@@ -1,6 +1,12 @@
 import { Text, TradeButton } from '@deriv/quill-design';
 import { MarketStatus } from '../types';
 
+const spreadClass = (status: MarketStatus) => {
+  return status === 'closed'
+    ? 'text-typography-disabled'
+    : 'text-typography-default';
+};
+
```
Comment (prince-deriv) [inline libs/components/src/lib/card/live-market/buy-sell.buttons.tsx:9]:
```suggestion
const spreadClassname = status === 'closed'  ? 'text-typography-disabled' : 'text-typography-default';

```

File: libs/components/src/lib/card/live-market/buy-sell.buttons.tsx [hunk 3]
```diff
@@ -11,10 +17,10 @@ export const BuySellButtons = ({
   return (
     <div className="flex flex-col gap-gap-md">
       <div className="flex flex-row gap-gap-md">
-        <Text size="sm" className="text-typography-default">
+        <Text size="sm" className={spreadClass(status)}>
```
Comment (prince-deriv) [inline libs/components/src/lib/card/live-market/buy-sell.buttons.tsx:20]:
```suggestion
        <Text size="sm" className={spreadClassname}>
```

File: libs/components/src/lib/card/live-market/buy-sell.buttons.tsx [hunk 4]
```diff
@@ -11,10 +17,10 @@ export const BuySellButtons = ({
...
           Spread
         </Text>
-        <Text size="sm" bold>
+        <Text size="sm" bold className={spreadClass(status)}>
```
Comment (prince-deriv) [inline libs/components/src/lib/card/live-market/buy-sell.buttons.tsx:23]: Same here

**Review by prince-deriv — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/card/live-market/index.tsx [hunk 5]
```diff
@@ -7,6 +7,7 @@ import {
 import { LiveMarketContent } from '../types';
 import LivePrice from './live-price';
 import { BuySellButtons } from './buy-sell.buttons';
+import clsx from 'clsx';
```
Comment (prince-deriv) [inline libs/components/src/lib/card/live-market/index.tsx:10]:
```suggestion

```

**Review by prince-deriv — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/card/live-market/index.tsx (hunk 5 above)
Comment (Nuzhy-Deriv) [reply libs/components/src/lib/card/live-market/index.tsx:10]: i'm using this `clsx` in line no 73

**Review by mohsen-deriv — APPROVED** [review]

**Pull request merged**